app/steps/ui/summary/includes/executors.njk

```diff
@@ -162,6 +177,18 @@
             alreadyDeclared = fields.summary.alreadyDeclared.value
         )}}
 
+        {{checkanswer(
+            items = [{
+                    question: content.Summary.currentNameReason,
+                    answer: executor.currentNameReason,
+                    visible: executor.hasOtherName
+                }
+            ],
+            url = content.ExecutorCurrentNameReason.url | replace("*", loop.index - 1 ),
```

Comment:
You can use `loop.index0` to access a zero-indexed array instead of `loop.index - 1`